db/migrate/20201022201901_update_amazon_tag_mapper_resource_type.rb

```diff
@@ -0,0 +1,14 @@
+class UpdateAmazonTagMapperResourceType < ActiveRecord::Migration[5.2]
+  class ProviderTagMapping < ActiveRecord::Base
+  end
+
+  def up
+    ProviderTagMapping.where(:labeled_resource_type => "Vm").update_all(:labeled_resource_type => "VmAmazon")
```

Comment:
Can you add a `say_with_time` on all of these?

Reply:
:+1: